arm/proofs/base.ml

```diff
@@ -54,6 +54,14 @@ extra_word_CONV :=
          SHA512SU1_REDUCE_CONV]
         @ (!extra_word_CONV);;
 
+(* ------------------------------------------------------------------------- *)
+(* Additional Cryptographic AES intrinsics                                   *)
+(* ------------------------------------------------------------------------- *)
+
+loadt "arm/proofs/aes.ml";;
+
+extra_word_CONV := [aese_CONV; aesmc_CONV] @ (!extra_word_CONV);;
```

Comment:
For `aese_CONV` and `aesmc_CONV`, should we follow the style of SHA256/SHA512 which starts with capital letters and explicitly describes that they are REDUCE?

Reply:
What is the difference between conversions that have REDUCE and ones that don't have?

Reply:
Done.